packages/helix/build.sh

```diff
@@ -15,30 +14,16 @@ termux_step_make_install() {
 
 	cargo build --jobs "${TERMUX_MAKE_PROCESSES}" --target "${CARGO_TARGET_NAME}" --release
 
-	cat > "hx" <<- EOF
-	#!${TERMUX_PREFIX}/bin/sh
+	cat >"hx" <<-EOF
+		#!${TERMUX_PREFIX}/bin/sh
 
-	HELIX_RUNTIME=${TERMUX_PREFIX}/lib/helix/runtime \\
-	exec ${TERMUX_PREFIX}/lib/helix/hx "\$@"
+		HELIX_RUNTIME=${TERMUX_PREFIX}/lib/helix/runtime \\
+		exec ${TERMUX_PREFIX}/lib/helix/hx "\$@"
 	EOF
-	install -Dm755 ./hx "${TERMUX_PREFIX}/bin/hx"
+	install -Dm744 ./hx "${TERMUX_PREFIX}/bin/hx"
 
-	mkdir -p "${TERMUX_PREFIX}/lib/helix"
+	install -Dm744 -t "${TERMUX_PREFIX}"/lib/helix target/"${CARGO_TARGET_NAME}"/release/hx
+	install -Dm644 -t "${TERMUX_PREFIX}"/lib/helix languages.toml
 
 	cp -r runtime "${TERMUX_PREFIX}/lib/helix"
-
-	install -Dm755 -t "${TERMUX_PREFIX}/lib/helix" "target/${CARGO_TARGET_NAME}/release/hx"
-	install -Dm666 -t "${TERMUX_PREFIX}/lib/helix" languages.toml
-}
-
-termux_step_create_debscripts() {
```

Comment:
Why removing this post install script?

Reply:
I don't think config file should be provided in package itself. User should create their own if needed.

BTW `~/.config/helix/*` files are used for overriding default, so makes sense not to provide default one.

Reply:
👍